kleva-j/ireporter-frontend-react-redux
# 164034018 Enable users to login into the application.

#### What does this PR do?
Implements the functionality for a user to log into the application.

#### Description of Task to be completed?
- setup required action for the redux store.
- create the folder structure for the components and pages.
- enable routing to the login page and the 404 pages.

#### How should this be manually tested?
- run `npm run start:dev`, this will launch the app and would load up the landing page in your default browser.
- navigate to this route `http://localhost:3000/login` to see the login page.
- enter your login credentials and click the login button.
- On successful login, you would be redirected to the dashboard.
- if any error occurs in the process, you would get a popup notification.

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
#164034018
